apps/fluent-tester/tsconfig.json

```diff
@@ -4,13 +4,10 @@
     "baseUrl": ".",
     "outDir": "lib",
     "paths": {
-      "*": [
-        "*",
-        "*.win32"
-      ]
-    }
+      "*": ["*", "*.win32", "./*"],
+      "src/*": ["./src/*", "src"]
+    },
+    "types": ["@wdio/sync", "@wdio/jasmine-framework", "node", "@types/jasmine"]
```

Comment:
I don't think listing the types here should be necessary- ts should be smart enough to look into node_modules/@types

Reply:
This comes directly from WebDriverIO's TypeScript implementation page. I will, however, remove them and see if it still works. Update coming

Reply:
Upon further testing, I fund these necessary. Removing the types causes errors when running the E2E tests.